---
title: 例外処理ステートメント (C# リファレンス)
ms.date: 07/20/2015
helpviewer_keywords:
- keywords [C#], exception handling
ms.assetid: 6d0323a3-3164-411c-9b84-a3606bd0e13c
ms.openlocfilehash: 1d48981e0d36053fce1249b27cd15551ed59dc91
ms.sourcegitcommit: ad99773e5e45068ce03b99518008397e1299e0d1
ms.translationtype: HT
ms.contentlocale: ja-JP
ms.lasthandoff: 09/22/2018
ms.locfileid: "46576945"
---
# <a name="exception-handling-statements-c-reference"></a>例外処理ステートメント (C# リファレンス)
C# では、プログラムの実行時に発生する例外と呼ばれる異常事態を処理する組み込みサポートが用意されています。 このような例外は、通常の制御フローの外部にあるコードによって処理されます。  
  
 ここでは、例外処理に関する次のトピックについて説明します。  
  
-   [throw](../../../csharp/language-reference/keywords/throw.md)  
  
-   [try-catch](../../../csharp/language-reference/keywords/try-catch.md)  
  
-   [try-finally](../../../csharp/language-reference/keywords/try-finally.md)  
  
-   [try-catch-finally](../../../csharp/language-reference/keywords/try-catch-finally.md)  
  
## <a name="see-also"></a>参照  

- [C# リファレンス](../../../csharp/language-reference/index.md)  
- [C# プログラミング ガイド](../../../csharp/programming-guide/index.md)  
- [C# のキーワード](../../../csharp/language-reference/keywords/index.md)  
- [ステートメントのキーワード](../../../csharp/language-reference/keywords/statement-keywords.md)  
- [例外と例外処理](../../../csharp/programming-guide/exceptions/index.md)
